src/UXCapture.js

```diff
@@ -46,6 +46,24 @@ const UXCapture = {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
 		_startMarkName = NAVIGATION_START_MARK_NAME;
+
+		if (config.useBrowserHistory) {
+			window.onpopstate = function(event) {
+				UXCapture.startTransition();
+			};
+
+			const pushState = window.history.pushState;
+			window.history.pushState = (...args) => {
+				UXCapture.startTransition();
```

Comment:
this might be assuming too much - e.g. `pushState` can be used for hash changes, which generally aren't implemented as view changes. We have some logic in `SyncContainer` that will only trigger location changes when there is a path change or querystring change, but not when there's another kind of 'navigation' https://github.com/meetup/meetup-web-platform/blob/hapi-signal-sciences/packages/mwp-router/src/SyncContainer.jsx#L29-L31

I'm wary of monkey-patching standard library APIs, so I think I would vote to keep this code change as a documented example of how a consumer library might integrate UXCapture with the History API rather than building it into the core